test/builtin/test_evaluation.py

```diff
@@ -62,6 +62,7 @@
             None,
         ),
         ("ClearAll[f];", None, None, None),
+        ("HoldForm[Times[]]", None, "Times[]", None),
```

Comment:
The case of formatting `Times[c]` was not handled before because it seems entangled with the format rule for Plus. I will fix this in a subsequent PR.

Reply:
Ok. Please commit off of this and we'll combine these together.